tests/kw_include_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+include "./tests/utils"
+
+function suite
+{
+  suite_addTest "include_Test"
+  suite_addTest "include_twice_Test"
+  suite_addTest "include_wrong_path_Test"
+}
+
+function include_Test()
+{
+  ID=1
+  include ./src/kwio.sh
+  expected_result="1"
```

Comment:
You don't need this variable; put it directly to the `assertEquals` (same this entire file).